packages/terra-clinical-detail-view/src/DetailView.jsx

```diff
@@ -35,19 +45,27 @@ const propTypes = {
   footer: PropTypes.string,
 
   /**
-   * Indicates if sections should be devided
+   * Indicates if sections should be divided
    */
   isDivided: PropTypes.bool,
+
+  /*
```

Comment:
Should be `/**` instead of `/*`
I just noticed this comment isn't showing as the description when generated on the example site.